app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
```

Comment:
1. `"rortsa"` tên biến chi chẳng hiểu vậy em? đặt cho có ý nghĩa nhé

2.  Chị không khởi tạo giá trị của `page_title = ""` được không?

3. Phân biệt cho chị empty?, nil?, blank? được ko?
Và sau đó cân nhắc xem dùng empty? đã hợp lý chưa nhé

Reply:
2. Dạ em ko khởi tạo giá trị page_title vẫn được, em sử dụng blank? để check xem thử page_title có được thực sự truyền vào hay không.
3. empty? và nil? được cung cấp bởi Ruby 
+ .empty? thường được sử dụng trên các Collection hoặc String (trả về true nếu length =0) 
+ .nil? trả về true nếu nó thật sự là nil (false.nil? vẫn trả về false :dancer: )

còn blank? được cung cấp bởi Rails
+ một object được coi là blank nếu nó nil, false, empty, hoặc là 1 chuỗi toàn các khoảng trắng
Như vậy em sử dụng page_title mà không khởi tạo giá trị cho nó và sử dụng .blank thì hợp lý (đpcm) :dancer: